_rest_routes.inc.php

```diff
@@ -431,6 +432,12 @@
         $data = (array) (json_decode(file_get_contents("php://input"), true));
         return (new FhirOrganizationRestController())->patch($id, $data);
     },
+    "GET /fhir/PractitionerRole" => function () {
+        return (new FhirPractitionerRoleRestController())->getAll($_GET);
+    },
+    "GET /fhir/PractitionerRole/:id" => function ($id) {
+        return (new FhirPractitionerRoleRestController())->getOne($id);
+    },
     "GET /fhir/AllergyIntolerance" => function () {
```

Comment:
also, add the following to the endpoints `"GET /fhir/Organization" => function () {` and `"GET /fhir/Organization/:id" => function ($id) {` endpoints:
```
RestConfig::authorization_check("admin", "users");
```